src/BenchmarksServer/Startup.cs

```diff
@@ -289,7 +273,7 @@ private static void DeleteDir(string path)
         private static Process StartProcess(string hostname, string benchmarksRepo, ServerJob job)
         {
             var filename = "dotnet";
-            var arguments = $"run -c Release -- --scenarios {job.Scenario} --server {job.WebHost} " +
+            var arguments = $"run -c Release -f netcoreapp1.1 -- --scenarios {job.Scenario} --server {job.WebHost} " +
```

Comment:
This particular change is a workaround for an extra / unused target framework in the Benchmarks project. Don't do this in this PR; either add that option or remove net451 from Benchmarks.csproj.